src/plugins/tlsmon/tlsmon.cpp

```diff
@@ -114,6 +114,7 @@
 #include "private.h"
 #include "tlsmon.h"
 
+#include "libusermode/uh-private.hpp"
```

Comment:
This doesn't look right, you should not have to include a library private header in a plugin.